Firebase/Auth/Source/Auth/FIRAuth.m

```diff
@@ -1884,19 +1884,12 @@ - (BOOL)getUser:(FIRUser *_Nullable *)outUser
     return [[FIRAuth alloc] initWithApp:container.app];
   };
   FIRComponent *authInterop = [FIRComponent componentWithProtocol:@protocol(FIRAuthInterop)
+                                              instantiationTiming:FIRInstantiationTimingAlwaysEager
```

Comment:
Why AlwaysEager?

Reply:
It mimics the previous behaviour removed on the previous line 1897:

 https://github.com/firebase/firebase-ios-sdk/pull/4137/files#diff-f33c561b2a3430b947e6da076758c6d9L1897

every time `[FIRApp configure]` is called, we called `[FIRAuth authWithApp:]` which instantiated the instance. 

The reason it's `AlwaysEager` and not `EagerInDefaultApp` is because `FIRAuth` works with multiple `FIRApp`s.

Reply:
Yeah, `FIRAuth` works with multiple `FIRApp` s.